eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MapIterable.java

```diff
@@ -95,6 +98,23 @@
      */
     void forEachKeyValue(Procedure2<? super K, ? super V> procedure);
 
+    /**
+     * Calls {@code injectInto} with each <em>key-value</em> pair of the map.
```

Comment:
I would rephrase this slightly to "Implements the injectInto pattern with each key-value..." since it doesn't actually call `injectInto`.